app/gui/src/controller/project.rs

```diff
@@ -135,7 +135,6 @@ impl Project {
 
         self.init_call_stack_from_metadata(&main_module_model, &main_graph).await;
         self.notify_about_compiling_process(&main_graph);
-        self.display_warning_on_unsupported_engine_version();
```

Comment:
Why do we do this? Shan't we rather update an appriopriate constant?

Reply:
sorry accidental commit